libs/@guardian/source-react-components-development-kitchen/src/toggle-switch/styles.ts

```diff
@@ -24,12 +24,10 @@ const toggleBorder = 'rgba(255, 255, 255, 0.6)';
  */
 const toggleBorderGreen = '#A7CFB8';
 
-export const buttonStyles = (
```

Comment:
Are these only used by ToggleSwitch?

Reply:
They’re inside the toggle-switch folder and typescript does not complain with the signature change and no visual regression, so seems safe to me!